cmake/GetGTSAM.cmake

```diff
@@ -1,9 +1,10 @@
 # CMake module to look for GTSAM
 # if it is not found then it is downloaded and marked for compilation and install
 
-find_package(GTSAM 4.3.0 EXACT QUIET)
+find_package(GTSAM 4.3.0 EXACT)
```

Comment:
Why remove the QUIET here. If GTSAM is not found CMake will now issue a warning, which is not necessary since we build our own in the next few lines.